supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc

```diff
@@ -83,18 +76,18 @@ For documentation questions, contact rhos-docs@redhat.com.
 [discrete]
 [[red-hat-openstack-platform]]
 ==== image:images/yes.png[yes] Red Hat OpenStack Platform (noun)
-*Description*: Spell out in full. This product name applies to Red Hat OpenStack Platform 8 and later versions.
+*Description*: On the first use in a module, use the complete product name and the abbreviation in parentheses: _Red Hat OpenStack Platform (RHOSP)_. After the first instance, use: _RHOSP_. This product name applies to RHOSP 8 and later versions.
```

Comment:
So a few months ago I came across something that said "a RHOSP" in the OpenShift docs and thought that it was wrong - I assumed it would be "an" because of saying out "R-H-O-S-P". But I was told that it was actually correct to use "a RHOSP" (I can't remember why). If this is true, I think it might be worthwhile to say here. wdyt?

Reply:
Correct, it should be pronounced "ROsp" or "RAHsp", whichever, and therefore "a RHOSP". This is the norm for other similar cases, like RHOSAK ("ROW-sack"), RHODS ("ROads"), RHOSR ("ROsr"), etc. However, that said, I can't imagine when or why we'd ever say "a RHOSP" anyway, except like "a RHOSP instance" or "a RHOSP [something]". So maybe just gloss the pronunciation if needed. Defer to @ndeevy whether or not address it.

Reply:
Suggest: "use _RHOSP_". Not sure why the colon.

Reply:
Thanks for the background @sterobin! To your point about not knowing when you'd use it, there is an instance of it in this PR:

```
Use director to create a RHOSP environment
```
But +1 to trying to rewrite to avoid the a/an issue, if possible.

Reply:
@bergerhoffer Ah, sorry, missed it! Yeah, that's a perfect example. So yeah, "a RHOSP environment" is correct/compliant.